src/member/factory.ts

```diff
@@ -1,21 +1,44 @@
-import { CompleteMember, MemberType } from './member.js';
+import {
+  Account,
+  BaseAccount,
+  CompleteGuest,
+  CompleteMember,
+  MemberType,
+} from './member.js';
 import { faker } from '@faker-js/faker';
 
+export function accountFactory(account: Partial<Account>): Account {
+  return { id: faker.string.uuid(), name: faker.person.fullName(), ...account };
+}
```

Comment:
@Defferrard That would help in the usage :)
```suggestion
export function AccountFactory(account: Partial<Account> = {}): Account {
  return { id: faker.string.uuid(), name: faker.person.fullName(), ...account };
}
```